src/include/sof/samples/audio/kwd_nn_detect_test.h

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_NXP_DETECT_TEST_H__
```

Comment:
USER? NXP? I mean it can be left here as an artefact but it would be best to update this one as well.